Mathlib/RingTheory/OreLocalization/Basic.lean

```diff
@@ -95,41 +92,49 @@ attribute [local instance] oreEqv
 variable {R S}
 variable {X} [MulAction R X]
 
-/-- The division in the ore localization `X[S⁻¹]`, as a fraction of an element of `X` and `S`. -/
+/-- The division in the Ore localization `X[S⁻¹]`, as a fraction of an element of `X` and `S`. -/
+@[to_additive "The subtraction in the Ore localization,
+as a difference of an element of `X` and `S`."]
 def oreDiv (r : X) (s : S) : X[S⁻¹] :=
   Quotient.mk' (r, s)
 #align ore_localization.ore_div OreLocalization.oreDiv
 
 @[inherit_doc]
 infixl:70 " /ₒ " => oreDiv
 
-@[elab_as_elim]
+@[to_additive (attr := elab_as_elim)]
 protected theorem ind {β : X[S⁻¹] → Prop}
     (c : ∀ (r : X) (s : S), β (r /ₒ s)) : ∀ q, β q := by
   apply Quotient.ind
   rintro ⟨r, s⟩
   exact c r s
 #align ore_localization.ind OreLocalization.ind
 
+@[to_additive]
 theorem oreDiv_eq_iff {r₁ r₂ : X} {s₁ s₂ : S} :
     r₁ /ₒ s₁ = r₂ /ₒ s₂ ↔ ∃ (u : S) (v : R), u • r₂ = v • r₁ ∧ u * s₂ = v * s₁ :=
   Quotient.eq''
 #align ore_localization.ore_div_eq_iff OreLocalization.oreDiv_eq_iff
 
 /-- A fraction `r /ₒ s` is equal to its expansion by an arbitrary factor `t` if `t * s ∈ S`. -/
+@[to_additive "A difference `r -ₒ s` is equal to its expansion by an
```

Comment:
Is this notation really defined? I have tried to use it with `open AddOreLocalization` and `open scoped AddOreLocalization`  but it did not work.

Reply:
Thanks! fixed.